architecture-building-systems/CityEnergyAnalyst
# New Schedules Maker / Model

This PR creates a new element in the workflow of CEA called the occupancy model.

We did the next changes:
1. We changed the occupancy_archetypes.xls to csv files. This allows us to track changes more easiliy.
2. We included one more option to the data helper. the user can now create occupancy schedules per building. An interface is being prepared by @reyery for this.
3. We included a new TOOL called "Building_schedules" This tool creates either a deterministic or stochaistic profile of occupancy and internal loads.

Now, before running the energy demand, you need to run the schedule-naker model first. 
the results are stored in outputs/data/occupancy.

The energy demand script uses these schedule_maker model files as input now.

to test:
1. run data helper with the new default configuration. This configuration includes an option to create archetypical daily schedules for every building (these are inputs of the schedule maker)
2. run the Building schedules tool (it is in the interface) under demand forecasting.
3. run the energy demand too. 

To verify, all should be passing the tests.
